### DISCUSSION TITLE
more premade models for other games i.e. battlefield 1/V/2042, arma, dayz, counter strike source, etc

### DISCUSSION THREAD

**Discussion by militaristica** [Babyhamsta/Aimmy#64] [Ideas]
shit take too long !!! please make us some pre-baked models to quickly load🙏

**Comment by whoswhip** [comment]
> shit take too long !!! please make us some pre-baked models to quickly load🙏

We now have a discord server full of models and with aimmy version 1.2-1.4 and above there is now a download models tab.

**Reply by AL99117** [reply]
Can you invite me to join?

**Comment by yurrii22** [comment]
bro im making a dayz one but i need ppl help me with pictures dayz needs alot of pics so if you wanna join me with one other member to make it happen just hit me up
